setup.py

```diff
@@ -24,6 +24,6 @@
         'wfa_cardinality_estimation_evaluation_framework',
         'wfa_cardinality_estimation_evaluation_framework.common',
         'wfa_cardinality_estimation_evaluation_framework.estimators',
+        'wfa_cardinality_estimation_evaluation_framework.evaluations',
         'wfa_cardinality_estimation_evaluation_framework.simulations',
```

Comment:
Probably also need to add 'wfa_cardinality_estimation_evaluation_framework.evaluations.data'?

Reply:
I don't think so.  We decided to keep the data module not as a package so you won't have to re-install the package every time you change code in data, as we expect data to change frequently

Reply:
my apologies, Preston pointed out a problem which I tracked to a misconfiguration of our github importer.  Hopefully the upcoming upload makes more sense